tools/labs/templates/hello_assignment/sol/Makefile

```diff
@@ -0,0 +1,13 @@
+KDIR=/usr/src/linux-so2
```

Comment:
The Makefile is not needed anymore, the top level makefile should take care of building